iocore/net/UnixNetAccept.cc

```diff
@@ -280,6 +234,14 @@ NetAccept::do_blocking_accept(EThread *t)
       return -1;
     }
 
+    // Throttle accepts
+    if (!backdoor && (check_net_throttle(ACCEPT, now) || net_memory_throttle)) {
+      Debug("net_accept", "Too many connections or too much memory used, throttling");
+      check_throttle_warning();
+      con.close();
```

Comment:
check_throttle_warning logs an error message.  There currently isn't a stat for throttled incoming connections.  I can file a bug for it since I believe it will need to be added in a couple places.